packages/core/admin/server/src/services/transfer/utils.ts

```diff
@@ -20,10 +13,17 @@ const hasValidTokenSalt = (): boolean => {
 /**
  * Checks whether data transfer features are enabled
  */
-const isDataTransferEnabled = (): boolean => {
+const isRemoteTransferEnabled = (): boolean => {
   const { utils } = getService('transfer');
 
-  return !utils.isDisabledFromEnv() && utils.hasValidTokenSalt();
+  // TODO v6: Remove this warning
+  if (env.bool('STRAPI_DISABLE_REMOTE_DATA_TRANSFER')) {
```

Comment:
Is this meant to check if the env variable is defined (regardless of the value), or if it has been manually disabled?

Reply:
I only cared about warning them if it's true (ie, if it's been disabled), but I'll make this a stricter check and warn simply for having the value in env since technically they should be aware either way.